src/chains/ethereum/options/src/chain-options.ts

```diff
@@ -196,7 +198,13 @@ export const ChainOptions: Definitions<ChainConfig> = {
   hardfork: {
     normalize,
     cliDescription: "Set the hardfork rules for the EVM.",
-    default: () => "london",
+    default: () => {
+      const forkArg = yargs.parse(process.argv)["fork"];
+      // If fork argument is blank, default value is mainnet
+      const fork =
+        !forkArg || forkArg.toString() == "true" ? "mainnet" : forkArg;
```

Comment:
What's up with checking `toString()` here? I assume the value would be either Boolean, or empty string (or an explicit value)?

It might be worthwhile making this check more explicit, for readability if nothing else.

Reply:
Actually there are 3 scenarios 
1. where --fork option is not provided 
2. --fork option without any value
3. --fork option with value
To cover second scenario where the value is not string I have added that change. Also yargs parse converts it to string. 
If I use process.argv directly then Boolean can be used, but it needs additional parsing logic to check all possibilites.